components/SectionTitle/index.tsx

```diff
@@ -0,0 +1,21 @@
+import * as React from 'react';
+import Styles from "./Section.module.css";
+
+export interface SectionTitleProps {
+    dataSection: Object;
+    title: String
+}
+
+export default function SectionTitle (props: SectionTitleProps) {
+  return (
+    <div>
+      <section className={Styles.Section_boxTitle}>
+        <div className={Styles.Section_Space}></div>
```

Comment:
```suggestion
        <div className={Styles.Section_Space} />
```
Nếu chỉ css đơn giản bác thử before after nhé